bin/countly.install_ubuntu.sh

```diff
@@ -52,15 +52,12 @@ apt-get -y install nginx || (echo "Failed to install nginx." ; exit)
 apt-get -y install nodejs || (echo "Failed to install nodejs." ; exit)
 
 #if npm is not installed, install it too
-if !(command -v npm >/dev/null) then
+if ! (command -v npm >/dev/null) then
     apt-get -y install npm
 fi
 
 #install supervisor
-if [ "$INSIDE_DOCKER" != "1" ]
-then
-	apt-get -y install supervisor || (echo "Failed to install supervisor." ; exit)
-fi
+apt-get -y install supervisor || (echo "Failed to install supervisor." ; exit)
```

Comment:
Regarding supervisor, I guess it was this one. It doesn't seem you use it though